### ISSUE TITLE
Add Tests?

### ISSUE THREAD

**Issue by awalsh3** [Contributees/First-Ruby-Quest#13]
Should we add tests to this project? I can add RSPEC if you think we should.

**Comment by leawhitelaw** [comment]
I'd be happy to work on adding tests with you!

**Comment by Oli0li** [comment]
Definitely! Feel free to start. I think the IssuesController might be a good place to start, I don't think we're going to make significant changes to it right away.

**Comment by leawhitelaw** [comment]
@Huiclaire and I are working through this currently - PR should be up soon.
